examples/krypta.rs

```diff
@@ -172,7 +182,8 @@ fn setup(
                     sprite: TextureAtlasSprite::new(id),
                     texture_atlas: floor_atlas.clone(),
                     ..default()
-                }).id());
+                })
+                .insert(RenderLayers::from_layers(CAMERA_LAYER_FLOOR)).id());
         }
```

Comment:
It could be worthwhile to use https://github.com/StarArawn/bevy_ecs_tilemap for the walls and floors of this example since I know that's eventually what I will be using and I've already seen at least 1 question about whether this stuff will work with it. I can take a stab at converting if that sounds good

Reply:
Unfortunately I don't support custom shaders, but this is making me want to revisit that..

Reply:
Yeah, we either can convert this one or add another example with tile-map. As long as there is a clear gain. I originally wanted to use LDTK plugin, but decided to keep examples to use as little external dependencies as possible.